tests/jest/pageScanner/context.test.js

```
@@ -0,0 +1,63 @@
+			</div>
+		`;
+
+		// Define context with exclude list matching src/pageScanner/index.js
+		const context = {
+			exclude: [ '#wpadminbar', '.edac-panel-container', '#query-monitor-main', '#qm-icon-container' ]
+		};
+
+		// Run axe with button-name rule to detect empty buttons
+		const results = await axe.run( context, {
+			runOnly: [ 'button-name' ]
+		} );
```

ESLint

[error] Missing trailing comma.

(comma-dangle)